recipe/bld.bat

```diff
@@ -150,9 +151,9 @@ if errorlevel 1 exit 1
 
 REM bytecode compile the standard library
 
-%PYTHON% -Wi %STDLIB_DIR%\compileall.py -f -q -x "bad_coding|badsyntax|py3_" %STDLIB_DIR%
+%PREFIX%\python.exe -Wi %PREFIX%\Lib\compileall.py -f -q -x "bad_coding|badsyntax|py3_" %PREFIX%\Lib
```

Comment:
After looking everything, this change seems fine as these paths all end up being the same in the end.